package.json

```diff
@@ -26,7 +27,8 @@
     "mktemp": "*",
     "nodemailer": "*",
     "nodemailer-smtp-transport": "*",
-    "country-data": "*"
+    "country-data": "*",
+    "openid-client": ">= 1.5.3"
```

Comment:
`"openid-client": "^1.5.3"` would be the best practice declaration of the version.